package.json

```diff
@@ -77,7 +77,7 @@
   },
   "homepage": "https://github.com/join-monster/join-monster#readme",
   "peerDependencies": {
-    "graphql": "^15.4.0"
+    "graphql": "^16.6.0|^15.4.0"
```

Comment:
Would `^16.0.0` work? So people on an older version of v16 aren't forced to upgrade. I think generally with peer deps, want to support as broad as possible.

Reply:
so 16.0.0 implies all subsequent versions of 16? I can take a quick look to see if the earlier versions have any issues. easy to test.

Reply:
ok. `^16.6.0` works. I tested `16.1.0` through `16.5.0` (and of course had already tested `16.6.0`).

Reply:
Does 16.0.0 work?